modules/javafx.graphics/src/test/java/test/com/sun/javafx/pgstub/StubFontLoader.java

```diff
@@ -48,8 +48,14 @@ public void loadFont(Font font) {
         StubFont nativeFont = new StubFont();
         nativeFont.font = font;
         String name = font.getName().trim().toLowerCase(Locale.ROOT);
-        if (name.equals("system") || name.equals("system regular")) {
+        if (name.equals("system regular")) {
             FontHelper.setNativeFont(font, nativeFont, font.getName(), "System", "Regular");
+        } else if (name.equals("system bold")) {
```

Comment:
does this mean this PR requires #1422 to be integrated first?

Reply:
Yes, that would be easier for reviewing this. Once merged, I will update this branch. :)